moe.karaokes.mugen.app.yml

```diff
@@ -0,0 +1,93 @@
+app-id: moe.karaokes.mugen.app
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
```

Comment:
```suggestion
base-version: '22.08'
```

Reply:
That actually causes no issues, so I'll have this commited, thank you. 

(I thought the base and runtime had to be aligned version wise)

Reply:
This actually does build but doesn't run if base is 22.08 but runtime is still 21.08

Reply:
> That actually causes no issues, so I'll have this commited, thank you.
> 
> (I thought the base and runtime had to be aligned version wise)

suggestion was to fix both. of course they have to match.

Reply:
Yes, my bad. We're working on it right now to update to the 22.08 runtime AND base.